apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableExpectationSamples.kt

```diff
@@ -0,0 +1,241 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class IterableExpectationSamples {
+    @Test
+    fun toContainBuilder() {
+        expect(listOf("A", "B"))
+            .toContain
+            .inOrder // order specifier
+            .only
+            .values("A", "B")
+
+        fails {
+            expect(listOf(5, 3, 2, 2, 4))
+                .toContain
+                .inAnyOrder // order specifier
+                .atMost(2)
+                .entries({ // assertion group about the entries
+                    toBeGreaterThan(2)
+                })
```

Comment:
Fine like this, you could also rewrite it to 
```suggestion
                .entry { // assertion group about the entries
                    toBeGreaterThan(2)
                }
```